doc/README.md

```diff
@@ -28,18 +28,18 @@ To produce the complete documentation in HTML, you will need Coq dependencies
 listed in [`INSTALL`](../INSTALL). Additionally, the Sphinx-based
 reference manual requires Python 3, and the following Python packages:
 
-    sphinx sphinx_rtd_theme beautifulsoup4 antlr4-python3-runtime pexpect sphinxcontrib-bibtex
+  - sphinx >= 1.8.0
```

Comment:
this seems to contradict what is in the docker file?

Reply:
Oops, thanks for noticing.